### PR TITLE
Allow Sparql endpoint to return no body.

### PR DESCRIPTION
**Pull request by JorgenEvens**
Allow empty response body as a result to update queries.

https://www.w3.org/TR/2013/REC-sparql11-http-rdf-update-20130321/#http-post

### CODE REVIEW

File: lib/client.js [hunk 1]
```diff
@@ -55,13 +55,22 @@ var SparqlClient = module.exports = function SparqlClient(endpoint, options) {
...
 
                 if (queryOptions) {
                     formatter.format(responseBody, queryOptions);
                 }
                 return promise.resolve(responseBody);
 
+                function parseJSON(body) {
+                    // Catch invalid JSON
+                    try {
+                        return JSON.parse(response.body);
```
Comment (eddieantonio) [inline lib/client.js:68]: By using `response.body`, `parseJSON(body)` discards its argument—which doesn't actually affect anything here, but still.

**Comment by eddieantonio** [comment]
@JorgenEvens Thanks for this PR! Yeah, I haven't had the time to carefully comb the SPARQL update spec for things like this (I mostly maintain this library for doing queries), so I appreciate you going out and linking to the spec here.

File: lib/client.js (hunk 1 above)
Comment (JorgenEvens) [inline lib/client.js:68]: Sorry about that, I'll try not to rush these PR's to much anymore and review them more carefully next time.

**Comment by JorgenEvens** [comment]
@eddieantonio Thanks for publishing these so quickly!

File: lib/client.js (hunk 1 above)
Comment (eddieantonio) [inline lib/client.js:68]: No worries! This is what code review is for! Catches tiny things like this.

**Pull request merged**